LabExT/Movement/Transformations.py

```diff
@@ -42,6 +44,124 @@ def stage_to_chip(self, stage_coordinate):
         pass
 
 
+class Coordinate(ABC):
+    """
+    Abstract base class of a coordinate with X, Y and Z values.
+
+    The base class cannot be initialised directly.
+    """
+    @classmethod
+    def from_list(cls, list: list) -> Type[Coordinate]:
+        """
+        Returns a new coordinate, created from a list.
+        """
+        return cls(*list[:3])
+
+    @classmethod
+    def from_numpy(cls, array: np.ndarray) -> Type[Coordinate]:
+        """
+        Returns a new coordinate created from a one-dimensional numpy array.
+        """
+        if array.ndim != 1:
+            raise ValueError("The given array is not a 1D-Array.")
+
+        return cls(*array.tolist()[:3])
```

Comment:
would it make sense to check for longer than 3 elem arrays? Or less than 3 elems?

Reply:
Arrays longer than 3 elements are simply truncated and shorter than 3 are padded with zeros.

Do you want to make a check that only arrays of length 3 are allowed?

Reply:
I dont see the padding code. But if you pad, then its ok.

Reply:
Okay, I see. thanks so much! Now just waiting on #48